index.html

```diff
@@ -248,23 +280,31 @@ <h2>About<br />Myself</h2>
             <div class="skill">
               <div class="skill-name">
                 <p class="name">Frameworks</p>
-                <a href="#"><img src="IMG/arrRight.png" alt="Arrow icon to expand section"/></a>
+                <a href="#"
+                  ><img
+                    src="IMG/arrRight.png"
+                    alt="Arrow icon to expand section"
+                /></a>
               </div>
             </div>
           </li>
           <li>
             <div class="skill">
               <div class="skill-name">
                 <p class="name">Skills</p>
-                <a href="#"><img src="IMG/arrRight.png" alt="Arrow icon to expand section" /></a>
+                <a href="#"
+                  ><img
+                    src="IMG/arrRight.png"
+                    alt="Arrow icon to expand section"
+                /></a>
               </div>
             </div>
           </li>
         </ul>
       </div>
     </section>
 
-    <section class="contact-section">
+    <section class="contact-section" id="contact">
       <form action="https://formspree.io/f/mayajwnb" method="post">
         <h2 class="contact-me-title">Contact Me</h2>
         <p class="contact-me-info">
```

Comment:
- [ ] Kindly note that your project has to closely follow the template design in Figma. Refer to the template and make the required changes. 
- The hamburger icon on the navigation bar is slightly different from the one in the design. The logo and menu icon should be horizontally aligned.
- The project cards have some of their styles scrambled. Kindly fix them.
- The contact form uses the wrong background colour and is missing the required background image decoration.
- You have a horizontal overflow on mobile screens.
_See the images below for more context._

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/122916467-c486b780-d311-11eb-8b53-13187bb6887b.png) | ![image](https://user-images.githubusercontent.com/49621279/122916594-e8e29400-d311-11eb-94a1-04cfcf148ebe.png)
 ![image](https://user-images.githubusercontent.com/49621279/122916782-1f201380-d312-11eb-91de-0b1f0bae1f9d.png) | ![image](https://user-images.githubusercontent.com/49621279/122916883-3c54e200-d312-11eb-849f-fd354993b279.png)
![image](https://user-images.githubusercontent.com/49621279/122916968-58588380-d312-11eb-870d-02572fae9b06.png) | ![image](https://user-images.githubusercontent.com/49621279/122917031-6c9c8080-d312-11eb-9e16-56ca02e8bf89.png)

Reply:
Not addressed